src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall(Ball ball) {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
```

Comment:
Should be class level fields
```suggestion
        ColorSupplier colorSupplier = new ColorSupplier();
        Random random = new Random();
```